mungegithub/mungers/submit-queue.go

```diff
@@ -950,6 +951,10 @@ func (sq *SubmitQueue) getGithubE2EStatus() []byte {
 	return sq.marshal(status)
 }
 
+func noMergeMessage(label string) string {
```

Comment:
FYI I like this one more because it is used in multiple places (and for testing)